dask/dataframe/io.py

```diff
@@ -450,6 +450,12 @@ def to_hdf(df, path_or_buf, key, mode='a', append=False, complevel=0,
     if '*' in key:
         single_node = False
 
+    if 'format' in kwargs:
```

Comment:
Sorry for being late. Why did we deleted those lines? Does this no longer apply?